.github/workflows/release.yml

```diff
@@ -2,8 +2,7 @@ name: Publish histolab to PyPI
 
 on:
   release:
-    branches:
-      - master
+    types: [released]
```

Comment:
could you elaborate?

Reply:
it was not valid actions yaml

<img width="315" alt="image" src="https://github.com/histolab/histolab/assets/20052362/1fefb20c-6a47-4c01-baad-72967505a12a">


https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release